proposals/lip-0030.md

```diff
@@ -377,7 +379,7 @@ accountMsg = {
 ```
 
 ```
-binaryMsg [307 bytes] = 0a14e11a11364738225813f86ea85214400e5db08d6e100a1a200fd3c50a6d3bd17ea806c0566cf6cf10f6e3697d9bda1820b00cb14746bcccef20052a4608021220c8b8fbe474a2b63ccb9744a409569b0a465ee1803f80435aec1c5e7fc2d4ee1812206115424fec0ce9c3bac5a81b5c782827d1f956fb95f1ccfa36c566d04e4d72673299010a180a07436174756c6c6f1201551820204028003080c6868f0112270a20cd32c73e9851c7137980063b8af64aa5a31651f8dcad258b682d2ddf091029e41080c2d72f12270a209d86ad24a3f030e5522b6598115bb4d70c1692c9d8995ddfccb377379a2d86c61080e59a771a2b0a20655e665765e3c42712d9a425b5b720d10457a5e45de0d4420e7c53ad73b02ef5108088debe01188001
+binaryMsg [307 bytes] = 0a14e11a11364738225813f86ea85214400e5db08d6e100a1805224608021220c8b8fbe474a2b63ccb9744a409569b0a465ee1803f80435aec1c5e7fc2d4ee1812206115424fec0ce9c3bac5a81b5c782827d1f956fb95f1ccfa36c566d04e4d72672a99010a180a07436174756c6c6f1201551820204028003080c6868f0112270a20c87ea0fa872534cd8fad11193960aa0e5579762af9c2bd23402b6d70fc62f2971080c2d72f12270a20248cd75f27b62a1db7c224c2dc1e2e74361f82085def3cb9b3dd371453f604f41080e59a771a2b0a20eac71db3f27d69de23919145f6eaa1af1512b2410d5c889d8a9eb0a585c2071c108088debe01188001
```

Comment:
shouldn't the length of the binaryMsg be shorter now?

Reply:
nice catch, thanks, fixed